asdl/gc_test.cc

```diff
@@ -44,7 +43,7 @@ TEST pretty_print_test() {
 GREATEST_MAIN_DEFS();
 
 int main(int argc, char** argv) {
-  gc_heap::gHeap.Init(1 << 20);
+  gHeap.Init(1 << 20);
```

Comment:
Hm I kind of like `gHeap` and `Obj` in a `gc_heap` namespace, but it's not that big a deal

On the other hand, Str,List,Dict are used by  generated code  (and duplicated)

Reply:
Yeah I was thinking it might make sense to namespace a couple _actual_ heap-related things like the allocator, etc.